operations/mimir/querier.libsonnet

```diff
@@ -23,6 +23,7 @@
       'querier.frontend-address': if !$._config.is_microservices_deployment_mode || $._config.query_scheduler_enabled then null else
         'query-frontend-discovery.%(namespace)s.svc.%(cluster_domain)s:9095' % $._config,
       'querier.frontend-client.grpc-max-send-msg-size': 100 << 20,
+      'querier.scheduler-client.grpc-max-send-msg-size': 100 << 20,
```

Comment:
"100 << 20" (which should be converted to human-readable number really), is the default value, and we don't need to set it. I'd suggest removing `querier.frontend-client.grpc-max-send-msg-size` instead (in separate PR).